example/Gruntconfig.json

```diff
@@ -5,8 +5,20 @@
   },
   "domain": "project.vm",
   "packages": {
-    "srcFiles": ["!sites/*/files/**", "!xmlrpc.php", "!modules/php/*"],
-    "projFiles": ["README*", "bin/**"]
+    "default": {
+      "srcFiles": ["!sites/*/files/**", "!xmlrpc.php", "!modules/php/*"],
+      "projFiles": ["README*", "bin/**"]
+    },
+    "acquia": {
+      "srcFiles": ["**", ".**", "!sites/default/**", "!sites/*/files/**", "!??/**", "!sites/all/modules/dev/**", "!sites/*/settings.php", "!sites/*/settings.db.inc", "!sites/*/settings.local.inc", "!.htaccess", "!.gitignore", "!.gitkeep"],
+      "sharedFiles": ["**", "!files/**", "!default.settings.php", "settings.php", "!settings.local.inc"],
+      "projFiles": ["README*", "bin/**"],
+      "root": "build/packages/acquia",
```

Comment:
I think "root" should be zero-config by appending the package "profile" key as a new path layer to the buildPaths.package config.